source/macos/AdaptiveCards/AdaptiveCards/Renderers/RichTextBlockRenderer.swift

```diff
@@ -71,7 +71,7 @@ class RichTextBlockRenderer: NSObject, BaseCardElementRendererProtocol {
             }
                     
             // apply underline to textrun
-            if textRun.getUnderline() {
+            if config.supportsSchemaVersion1Dot3, textRun.getUnderline() {
                 textRunContent.addAttributes([.underlineStyle: NSUnderlineStyle.single.rawValue], range: NSRange(location: 0, length: textRunContent.length))
             }
```

Comment:
Does Windows support this? If they do we should remove this check to maintain parity.

Reply:
I checked with ram, he told they don't